sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerClient.java

```diff
@@ -355,10 +322,7 @@
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
     public SyncPoller<OperationResult, List<RecognizedReceipt>>
-        beginRecognizeReceipts(InputStream receipt, long length, FormContentType formContentType,
-        boolean includeTextDetails, Duration pollInterval) {
-        Flux<ByteBuffer> buffer = Utility.toFluxByteBuffer(receipt);
-        return client.beginRecognizeReceipts(buffer, length, formContentType, includeTextDetails, pollInterval)
-            .getSyncPoller();
+        beginRecognizeReceipts(RecognizeOptions recognizeOptions) {
```

Comment:
Should this be named `BeginRecognizeReceiptsOptions` as per [guidelines](https://azure.github.io/azure-sdk/java_design.html#service-method-parameters)?

>DO name the options type after the name of the service method it is used for, such that the type is named <operation>Options. For example, above the method was createBlob, and so the options type was named CreateBlobOptions

Reply:
I think this could serve as an exception since `RecognizeOPtions` is not confined to just receipts API. It can be applied to the `receipt` and `content` API.